core/db/migrate/20250225051308_add_vat_id_email_and_reverse_charge_status_to_addresses.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddVatIdEmailAndReverseChargeStatusToAddresses < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_addresses, :vat_id, :string
```

Comment:
Sorry for the late comment, but is `vat_id` the right name internationally? Would `tax_id` be a better name, or is `vat_id` widely known? Draining some of you deep knowledge @fthobe

Reply:
Vat and tax id are different, though in many jurisdictions they have the same value. Vat id for (reverse charge) vat, tax id is usually the domestic tax number. Germany is one of the countries in which the number is not identical, in Italy it is.

Reply:
Thanks. Will leave it to `vat_id` 👍🏻